hack/run-e2e-tests-using-kubectl-kind.sh

```diff
@@ -14,7 +12,8 @@ fi
 kubectl patch clusterrolebinding cluster-admin --type='json' -p='[{"op": "add", "path": "/subjects/1", "value": {"kind": "ServiceAccount", "name": "default", "namespace": "default" } }]'
 kubectl run test-pod --env="HUMIO_E2E_LICENSE=$HUMIO_E2E_LICENSE" --env="E2E_LOGS_HUMIO_HOSTNAME=$E2E_LOGS_HUMIO_HOSTNAME" --env="E2E_LOGS_HUMIO_INGEST_TOKEN=$E2E_LOGS_HUMIO_INGEST_TOKEN" --env="E2E_RUN_ID=$E2E_RUN_ID" --env="GINKGO_NODES=$GINKGO_NODES" --env="DOCKER_USERNAME=$DOCKER_USERNAME" --env="DOCKER_PASSWORD=$DOCKER_PASSWORD" --restart=Never --image=testcontainer --image-pull-policy=Never -- sleep 86400
 while [[ $(kubectl get pods test-pod -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do echo "waiting for pod" ; kubectl describe pod test-pod ; sleep 1 ; done
-kubectl exec test-pod -- hack/run-e2e-tests-kind.sh
 
-end=$(date +%s)
-echo "Running e2e tests with kubectl exec took $((end-start)) seconds"
```

Comment:
Couldn't we just capture the exit-code with `KUBECTL_EXIT_CODE=$?` and then just end the script with `exit $KUBECTL_EXIT_CODE`.